java/java-tests/testSrc/com/intellij/psi/formatter/java/JavaFormatterAlignmentTest.java

```diff
@@ -49,6 +49,20 @@ public void testMethodAndChainedField() throws Exception {
     );
   }
 
+  public void testChainedMethodWithComments() throws Exception {
+    getSettings().ALIGN_MULTILINE_CHAINED_METHODS = true;
+    doMethodTest("AA.bb()\n" +
+                 ".cc() // comment after line\n" +
+                 ".dd()\n " +
+                 "        /* block comment on empty line */\n" +
```

Comment:
I agree that it's confusing. It's because the test wraps this in a class and method before formatting, indents all lines, except the comments. So I had to indent that one manually, to start at the same place as the method body would. Will provide you screenshots (and probably push another change that explains this in the comment itself)